pyomo/contrib/parmest/examples/reactor_design/updatesuffix_example.py

```diff
@@ -0,0 +1,67 @@
+#  ___________________________________________________________________________
```

Comment:
Second grump-inducing comment. In terms of screen reader accessibility, `updatesuffix` isn't the best. There should be an underscore between them: `update_suffix`. I recommend changing the filename to `update_suffix_example.py`